helm/single-content/values.yaml

```diff
@@ -25,7 +25,9 @@ idolImage:
   registry: "microfocusidolserver"
   # -- used to construct container image name: {idolImage.registry}/{idolImage.repo}:{idolImage.version}
   version: "23.4"
-  # -- used to construct container image name: {idolImage.registry}/{idolImage.repo}:{idolImage.version}
+  # -- used to construct container image name: {idolImage.registry}/{idolImage.repo}:{idolImage.version}.
+  #    If "agentstore" is specified then resources/agentstore.cfg will be used as the content engine's
+  #    configuration file.
   repo: "content"
```

Comment:
I don't think I like overloading the use of `idolImage.repo`. I'd probably go for adding a separate value like `isAgentstore: true` and use that to control the extra behaviour.